.github/container/Dockerfile.jax

```diff
@@ -55,13 +55,21 @@ ARG BUILD_DATE
 
 ENV BUILD_DATE=${BUILD_DATE}
 # The following environment variables tune performance
-# TODO: --xla_gpu_enable_xla_runtime_executable=true should be removed ASAP. It is a WAR for a crash observed in the default runtime leading to this error message: "nccl_all_reduce_thunk.cc:175] Check failed: reduction_kind.has_value()"
-# TODO: --xla_gpu_enable_llvm_module_compilation_parallelism=true should be removed ASAP. It is a WAR for the following LLVM crash observed in Pax: "Invalid LLVM IR before optimizations"
-ENV XLA_FLAGS="--xla_gpu_enable_latency_hiding_scheduler=true --xla_gpu_enable_async_all_gather=true --xla_gpu_enable_async_reduce_scatter=true --xla_gpu_enable_triton_gemm=false --xla_gpu_enable_xla_runtime_executable=true --xla_gpu_enable_llvm_module_compilation_parallelism=true"
+ENV XLA_FLAGS=""
+ENV XLA_FLAGS="${XLA_FLAGS} --xla_gpu_enable_latency_hiding_scheduler=true"
+ENV XLA_FLAGS="${XLA_FLAGS} --xla_gpu_enable_async_all_gather=true"
+ENV XLA_FLAGS="${XLA_FLAGS} --xla_gpu_enable_async_reduce_scatter=true"
+ENV XLA_FLAGS="${XLA_FLAGS} --xla_gpu_enable_triton_gemm=false"
 ENV CUDA_DEVICE_MAX_CONNECTIONS=1
 ENV NCCL_IB_SL=1
 ENV NCCL_NVLS_ENABLE=0
 ENV CUDA_MODULE_LOADING=EAGER
+## The following environment variables are WARs for known issues
+## Remove ASAP once the underlying issue has been removed
+# https://github.com/NVIDIA/JAX-Toolbox/pull/472, WAR for a crash observed in the default runtime leading to this error message: "nccl_all_reduce_thunk.cc:175] Check failed: reduction_kind.has_value()"
+ENV XLA_FLAGS="${XLA_FLAGS} --xla_gpu_enable_xla_runtime_executable=true"
+# https://github.com/NVIDIA/JAX-Toolbox/pull/487, WAR for the following LLVM crash observed in Pax: "Invalid LLVM IR before optimizations"
+ENV XLA_FLAGS="${XLA_FLAGS} --xla_gpu_enable_llvm_module_compilation_parallelism=true"
```

Comment:
`--xla_gpu_enable_llvm_module_compilation_parallelism=true` has been removed in upstream JAX-toolbox. Can we rebase this PR?

Reply:
Sure.

Reply:
I just rebased. Could you please examine if the flags are up-to-date now?

Reply:
Looks good, thanks!